processor/labelsprocessor/README.md

```diff
@@ -0,0 +1,23 @@
+# Labels Processor
+
+Supported pipeline types: metrics
+
+The labels processor can be used to add data point labels to all metrics that pass through it.
+If any specified labels already exist in the metric, the value will be updated.
+
+Please refer to [config.go](./config.go) for the config spec.
+
+Example:
+
+```yaml
+processors:
+  labels_processor:
+    labels:
+      - key: label1
+        value: value1
+      - key: label2
+        value: value2
```

Comment:
Maybe add an example for the environment variable labels and add a link to how you can configure environment variables with yaml files (like how you can set defaults values, etc.)

Reply:
The next few lines of code after this is an example with environment variables

Reply:
oops didnt realise this was in the README. I did link the config file below which shows more detailed example usage though